### PR TITLE
Microsoft-DHCP-Modeling-Rules

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created to provide modeling rules for Microsoft DHCP.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

File: Packs/MicrosoftDHCP/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Microsoft DHCP
+
+This pack includes Cortex XSIAM content.
+
+## Configuration on Server Side
+
+1. Start the DHCP administration tool (go to Start, Programs, Administrative Tools, and click **DHCP**).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDHCP/README.md:7]:
```suggestion
1. Start the DHCP administration tool (go to Start > Programs >Administrative Tools, and click **DHCP**).
```

File: Packs/MicrosoftDHCP/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+2. Right-click the DHCP server, and select **Properties** from the context menu.
+3. Select the **General** tab.
+4. Select the "Enable DHCP audit logging" check box.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDHCP/README.md:10]:
```suggestion
4. Select the **Enable DHCP audit logging** checkbox.
```

File: Packs/MicrosoftDHCP/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+5. Click OK.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDHCP/README.md:11]:
```suggestion
5. Click **OK**.
```

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

**Comment by nkanon** [comment]
> @nkanon Doc review completed.

As always, Thank a lot @ShirleyDenkberg !

**Review by evisochek — APPROVED** [review]

**Pull request merged**